plugins/aggregators/derivative/README.md

```diff
@@ -3,18 +3,69 @@
 The Derivative Aggregator Plugin estimates the derivative for all fields of the
 aggregated metrics.
 
+## Configuration
+
+```toml @sample.conf
+# Calculates a derivative for every field.
+[[aggregators.derivative]]
+  ## The period in which to flush the aggregator.
+  period = "30s"
+  ##
+  ## Suffix to append for the resulting derivative field.
+  # suffix = "_rate"
+  ##
+  ## Field to use for the quotient when computing the derivative.
+  ## When using a field as the derivation parameter the name of that field will
+  ## be used for the resulting derivative, e.g. *fieldname_by_parameter*.
+  ## By default the timestamps of the metrics are used and the suffix is omitted.
+  # variable = ""
+  ##
+  ## Maximum number of roll-overs in case only one measurement is found during a period.
+  # max_roll_over = 10
+```
+
+This aggregator will estimate a derivative for each field of a metric, which is
+contained in both the first and last metric of the aggregation interval.
+Without further configuration the derivative will be calculated with respect to
+the time difference between these two measurements in seconds.
+The following formula is applied is for every field
+$$
+derivative = {{value}_{last} - {value}_{first} \over {{time}_{last} - {time}_{first}}}
+$$
```

Comment:
This isn't being formatted on github, maybe because it needs to be a separate block with blank lines before and after.

I'm not sold on using LaTeX math in the readme files in general. It looks very nice and professional when it's formatted on github, but it is less readable in text. Also, the readme files are copied/embedded into InfluxDB cloud 2 using a different markdown renderer and there's a good chance it doesn't have a LaTeX formatter.

Reply:
Done.